src/components/views/elements/KeyboardShortcut.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+Copyright 2021 Šimon Brandner <simon.bra.ag@gmail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import * as React from "react";
+import { _td } from "../../../languageHandler";
+import {isMac, Key, Modifiers} from "../../../Keyboard";
+import {KeyCombo} from "../../../KeyBindingsManager";
+
+const keyName: Record<string, string> = {
```

Comment:
can this not be `Record<Key, string>`?

Reply:
I'd like it to be that, but since `Key` is an enum it won't work because there would need to be an entry in the `Record` for each `Key`. I might be able to use a Map though.